src/table/operations.jl

```diff
@@ -0,0 +1,60 @@
+import Base: filter, map, reduce
+
+"""
+    map(f, d::DTableRows)
+
+Applies `f` to each row of `d`.
+
+# Examples
+```jldoctest        
+    d = DTable((a=rand(1000), b=rand(1000)), chunksize=100)
+    m = map(x -> x.a + x.b, eachrow(d))
+    fetch(m)
+```
+"""
+function map(f, d::DTableRows)
+    chunk_wrap = chunk -> map(f, eachrow(chunk))
+    result_vector = [Dagger.@spawn chunk_wrap(c) for c in d.dtable.chunks]
+    Dagger.@spawn vcat(result_vector...)
+end
+
+function map(f, d::DTableColumns)
+    throw("Not implemented")
+end
+
+"""
+    reduce(f_reduce::Function, f_row::Function, d::DTableRows; init)
```

Comment:
```suggestion
    reduce(f_reduce::Function, f_row::Function, d::DTableRows; init) -> DTable
```

Reply:
Doesn't return a DTable if that's the meaning